editor/state.js

```diff
@@ -12,6 +12,8 @@ import { reduce, keyBy, first, last, omit, without, flowRight } from 'lodash';
 import { combineUndoableReducers } from './utils/undoable-reducer';
 import effects from './effects';
 
+const isMobile = window.innerWidth <= 782;
```

Comment:
Ever so minor: [The breakpoint](https://github.com/WordPress/gutenberg/blob/392333661d5d7334db722ce339520b78dcef25e8/editor/assets/stylesheets/_mixins.scss#L19-L23) will consider a viewport width of 782 as medium, not small. More accurately this should be `<`, not `<=`.